map/map_loader/config/lanelet2_map_loader.param.yaml

```diff
@@ -1,6 +1,12 @@
 /**:
   ros__parameters:
+    enable_differential_map_loading: $(var enable_differential_map_loading)      # flag to enable loading of differential map
     allow_unsupported_version: true             # flag to load unsupported format_version anyway. If true, just prints warning.
     center_line_resolution: 5.0                 # [m]
     use_waypoints: true                         # "centerline" in the Lanelet2 map will be used as a "waypoints" tag.
     lanelet2_map_path: $(var lanelet2_map_path) # The lanelet2 map path
+    dummy_metadata:                             # When the dynamic map loading is true and the map is a single file, this parameter will generate the metadata.
```

Comment:
I don't think this has to be a parameter.

Reply:
Actually, we are requiring meta_data_path when differential loading is turned on, so we don't need to generate dummy_data in the first place.